pkg/controller/operators/olm/operator_test.go

```diff
@@ -163,15 +164,22 @@ func NewFakeOperator(clientObjs []runtime.Object, k8sObjs []runtime.Object, extO
 
 	// Create the new operator
 	queueOperator, err := queueinformer.NewOperatorFromClient(opClientFake, logrus.StandardLogger())
+
+	lister := operatorlister.NewLister()
+	csvSetGenerator := csvutility.NewSetGenerator(logrus.StandardLogger(), lister)
```

Comment:
if you're passing in the logger here, you should construct it ahead of time and pass the same instance to `NewOperator`, `NewSetGenerator`, and `NewReplaceFinder` so that when we pass in configuration (i.e. in the e2e tests) it gets distributed everywhere

Reply:
sounds good, I will make the change. I noticed that `logrus.StandardLogger()` was being called  every time for a logger inside the function so went ahead with that.